[2025-05-25T17:35:38.082+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_pipeline.create_schema scheduled__2025-05-01T00:00:00+00:00 [queued]>
[2025-05-25T17:35:38.097+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_pipeline.create_schema scheduled__2025-05-01T00:00:00+00:00 [queued]>
[2025-05-25T17:35:38.097+0000] {taskinstance.py:1331} INFO - Starting attempt 1 of 2
[2025-05-25T17:35:38.116+0000] {taskinstance.py:1350} INFO - Executing <Task(PythonOperator): create_schema> on 2025-05-01 00:00:00+00:00
[2025-05-25T17:35:38.126+0000] {standard_task_runner.py:57} INFO - Started process 808 to run task
[2025-05-25T17:35:38.134+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'data_pipeline', 'create_schema', 'scheduled__2025-05-01T00:00:00+00:00', '--job-id', '18', '--raw', '--subdir', 'DAGS_FOLDER/data_pipeline.py', '--cfg-path', '/tmp/tmp1jzc2uk1']
[2025-05-25T17:35:38.139+0000] {standard_task_runner.py:85} INFO - Job 18: Subtask create_schema
[2025-05-25T17:35:38.249+0000] {task_command.py:410} INFO - Running <TaskInstance: data_pipeline.create_schema scheduled__2025-05-01T00:00:00+00:00 [running]> on host bf217478d96c
[2025-05-25T17:35:38.374+0000] {taskinstance.py:1568} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='data_pipeline' AIRFLOW_CTX_TASK_ID='create_schema' AIRFLOW_CTX_EXECUTION_DATE='2025-05-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-05-01T00:00:00+00:00'
[2025-05-25T17:35:38.377+0000] {taskinstance.py:1847} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/operators/python.py", line 181, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/operators/python.py", line 198, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/data_pipeline.py", line 40, in create_schema
    engine = create_engine(RAW_DB_URI)
NameError: name 'create_engine' is not defined
[2025-05-25T17:35:38.401+0000] {taskinstance.py:1368} INFO - Marking task as UP_FOR_RETRY. dag_id=data_pipeline, task_id=create_schema, execution_date=20250501T000000, start_date=20250525T173538, end_date=20250525T173538
[2025-05-25T17:35:38.418+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 18 for task create_schema (name 'create_engine' is not defined; 808)
[2025-05-25T17:35:38.466+0000] {local_task_job_runner.py:232} INFO - Task exited with return code 1
[2025-05-25T17:35:38.526+0000] {taskinstance.py:2674} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-05-25T17:38:42.944+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_pipeline.create_schema scheduled__2025-05-01T00:00:00+00:00 [queued]>
[2025-05-25T17:38:42.969+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_pipeline.create_schema scheduled__2025-05-01T00:00:00+00:00 [queued]>
[2025-05-25T17:38:42.971+0000] {taskinstance.py:1331} INFO - Starting attempt 1 of 2
[2025-05-25T17:38:42.995+0000] {taskinstance.py:1350} INFO - Executing <Task(PythonOperator): create_schema> on 2025-05-01 00:00:00+00:00
[2025-05-25T17:38:43.007+0000] {standard_task_runner.py:57} INFO - Started process 852 to run task
[2025-05-25T17:38:43.018+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'data_pipeline', 'create_schema', 'scheduled__2025-05-01T00:00:00+00:00', '--job-id', '19', '--raw', '--subdir', 'DAGS_FOLDER/data_pipeline.py', '--cfg-path', '/tmp/tmp7jfsx1j7']
[2025-05-25T17:38:43.022+0000] {standard_task_runner.py:85} INFO - Job 19: Subtask create_schema
[2025-05-25T17:38:43.152+0000] {task_command.py:410} INFO - Running <TaskInstance: data_pipeline.create_schema scheduled__2025-05-01T00:00:00+00:00 [running]> on host bf217478d96c
[2025-05-25T17:38:43.312+0000] {taskinstance.py:1568} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='data_pipeline' AIRFLOW_CTX_TASK_ID='create_schema' AIRFLOW_CTX_EXECUTION_DATE='2025-05-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-05-01T00:00:00+00:00'
[2025-05-25T17:38:43.327+0000] {python.py:183} INFO - Done. Returned value was: None
[2025-05-25T17:38:43.350+0000] {taskinstance.py:1368} INFO - Marking task as SUCCESS. dag_id=data_pipeline, task_id=create_schema, execution_date=20250501T000000, start_date=20250525T173842, end_date=20250525T173843
[2025-05-25T17:38:43.389+0000] {local_task_job_runner.py:232} INFO - Task exited with return code 0
[2025-05-25T17:38:43.449+0000] {taskinstance.py:2674} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-05-25T18:28:38.305+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_pipeline.create_schema scheduled__2025-05-01T00:00:00+00:00 [queued]>
[2025-05-25T18:28:38.326+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_pipeline.create_schema scheduled__2025-05-01T00:00:00+00:00 [queued]>
[2025-05-25T18:28:38.327+0000] {taskinstance.py:1331} INFO - Starting attempt 1 of 2
[2025-05-25T18:28:38.349+0000] {taskinstance.py:1350} INFO - Executing <Task(PythonOperator): create_schema> on 2025-05-01 00:00:00+00:00
[2025-05-25T18:28:38.361+0000] {standard_task_runner.py:57} INFO - Started process 89 to run task
[2025-05-25T18:28:38.371+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'data_pipeline', 'create_schema', 'scheduled__2025-05-01T00:00:00+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/data_pipeline.py', '--cfg-path', '/tmp/tmpmialzb0e']
[2025-05-25T18:28:38.379+0000] {standard_task_runner.py:85} INFO - Job 3: Subtask create_schema
[2025-05-25T18:28:38.478+0000] {task_command.py:410} INFO - Running <TaskInstance: data_pipeline.create_schema scheduled__2025-05-01T00:00:00+00:00 [running]> on host 13c2d9b41bb1
[2025-05-25T18:28:38.645+0000] {taskinstance.py:1568} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='data_pipeline' AIRFLOW_CTX_TASK_ID='create_schema' AIRFLOW_CTX_EXECUTION_DATE='2025-05-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-05-01T00:00:00+00:00'
[2025-05-25T18:28:38.664+0000] {python.py:183} INFO - Done. Returned value was: None
[2025-05-25T18:28:38.699+0000] {taskinstance.py:1368} INFO - Marking task as SUCCESS. dag_id=data_pipeline, task_id=create_schema, execution_date=20250501T000000, start_date=20250525T182838, end_date=20250525T182838
[2025-05-25T18:28:38.739+0000] {local_task_job_runner.py:232} INFO - Task exited with return code 0
[2025-05-25T18:28:38.794+0000] {taskinstance.py:2674} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-05-25T19:22:38.877+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_pipeline.create_schema scheduled__2025-05-01T00:00:00+00:00 [queued]>
[2025-05-25T19:22:38.896+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_pipeline.create_schema scheduled__2025-05-01T00:00:00+00:00 [queued]>
[2025-05-25T19:22:38.897+0000] {taskinstance.py:1331} INFO - Starting attempt 1 of 2
[2025-05-25T19:22:38.919+0000] {taskinstance.py:1350} INFO - Executing <Task(PythonOperator): create_schema> on 2025-05-01 00:00:00+00:00
[2025-05-25T19:22:38.931+0000] {standard_task_runner.py:57} INFO - Started process 75 to run task
[2025-05-25T19:22:38.947+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'data_pipeline', 'create_schema', 'scheduled__2025-05-01T00:00:00+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/data_pipeline.py', '--cfg-path', '/tmp/tmpasjx8up8']
[2025-05-25T19:22:38.965+0000] {standard_task_runner.py:85} INFO - Job 3: Subtask create_schema
[2025-05-25T19:22:39.083+0000] {task_command.py:410} INFO - Running <TaskInstance: data_pipeline.create_schema scheduled__2025-05-01T00:00:00+00:00 [running]> on host 65cea7b09e55
[2025-05-25T19:22:39.221+0000] {taskinstance.py:1568} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='data_pipeline' AIRFLOW_CTX_TASK_ID='create_schema' AIRFLOW_CTX_EXECUTION_DATE='2025-05-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-05-01T00:00:00+00:00'
[2025-05-25T19:22:39.242+0000] {python.py:183} INFO - Done. Returned value was: None
[2025-05-25T19:22:39.259+0000] {taskinstance.py:1368} INFO - Marking task as SUCCESS. dag_id=data_pipeline, task_id=create_schema, execution_date=20250501T000000, start_date=20250525T192238, end_date=20250525T192239
[2025-05-25T19:22:39.316+0000] {local_task_job_runner.py:232} INFO - Task exited with return code 0
[2025-05-25T19:22:39.369+0000] {taskinstance.py:2674} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-05-25T19:41:00.670+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_pipeline.create_schema scheduled__2025-05-01T00:00:00+00:00 [queued]>
[2025-05-25T19:41:00.694+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_pipeline.create_schema scheduled__2025-05-01T00:00:00+00:00 [queued]>
[2025-05-25T19:41:00.695+0000] {taskinstance.py:1331} INFO - Starting attempt 1 of 2
[2025-05-25T19:41:00.713+0000] {taskinstance.py:1350} INFO - Executing <Task(PythonOperator): create_schema> on 2025-05-01 00:00:00+00:00
[2025-05-25T19:41:00.724+0000] {standard_task_runner.py:57} INFO - Started process 315 to run task
[2025-05-25T19:41:00.736+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'data_pipeline', 'create_schema', 'scheduled__2025-05-01T00:00:00+00:00', '--job-id', '5', '--raw', '--subdir', 'DAGS_FOLDER/data_pipeline.py', '--cfg-path', '/tmp/tmp6r84faed']
[2025-05-25T19:41:00.741+0000] {standard_task_runner.py:85} INFO - Job 5: Subtask create_schema
[2025-05-25T19:41:00.843+0000] {task_command.py:410} INFO - Running <TaskInstance: data_pipeline.create_schema scheduled__2025-05-01T00:00:00+00:00 [running]> on host 65cea7b09e55
[2025-05-25T19:41:00.976+0000] {taskinstance.py:1568} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='data_pipeline' AIRFLOW_CTX_TASK_ID='create_schema' AIRFLOW_CTX_EXECUTION_DATE='2025-05-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-05-01T00:00:00+00:00'
[2025-05-25T19:41:00.993+0000] {python.py:183} INFO - Done. Returned value was: None
[2025-05-25T19:41:01.013+0000] {taskinstance.py:1368} INFO - Marking task as SUCCESS. dag_id=data_pipeline, task_id=create_schema, execution_date=20250501T000000, start_date=20250525T194100, end_date=20250525T194101
[2025-05-25T19:41:01.064+0000] {local_task_job_runner.py:232} INFO - Task exited with return code 0
[2025-05-25T19:41:01.113+0000] {taskinstance.py:2674} INFO - 1 downstream tasks scheduled from follow-on schedule check
